problemtools/verifyproblem.py

```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/env python2
+#! /usr/bin/env python3
 # -*- coding: utf-8 -*-
 from __future__ import print_function
```

Comment:
The `__future__` import would be made redundant by this (here and elsewhere).

Reply:
Fixed, thanks.